monero-outreach-docs/translations/es/archive/dandelion_es.md

```diff
@@ -0,0 +1,68 @@
+
+Existen pequeñas desventajas y limitaciones. Dandelion++ añade retrasos en la propagación de las transacciones. Esto tiene múltiples causas, que son discutidas por los mismos autores [3]. Primero, la existencia de la fase de tallo causa posiblemente retrasos de algunos segundos, basados en análisis y experimentaciones evaluadas por los mismos autores. La difusión, normalmente añade retrasos aleatorios de menos de un segundo durante la fase de cipsela. Si la red llega a estar sometida a un ataque black-hole, la confirmación de espera por el iniciador podría causar retrazos por algunos minutos en la transmisión. No se esperan que estos retrazos afecten la red. Además, Dandelion++ no encripta los paquetes P2P y no protege en contra del espionaje a nivel de ISP/VPN (Proveedor de servicio de Internet/Red Privada Virtual). Para esto utiliza Tor.
+
+[1] Exploring the Monero Peer-to-Peer Network, T. Cao, J. Yu, J. Decouchant, X Luo, and P. Verissimo, [eprint.iacr.org/2019/411.pdf](https://eprint.iacr.org/2019/411.pdf).  
+[2] Dandelion: Redesigning the Bitcoin Network for Anonymity, S.B. Venkatakrishan, G. Fanti, and P. Viswanath, SIGMETRICS ’17, June 5-9, 2017, Urbana-Champaign, [publish.illinois.edu/science-of-security-lablet/files/2016/07/Dandelion-Redesigning-BitCoin-Networking-for-Anonymity.pdf](http://publish.illinois.edu/science-of-security-lablet/files/2016/07/Dandelion-Redesigning-BitCoin-Networking-for-Anonymity.pdf)  
+[3] Dandelion++: Lightweight Cryptocurrency Networking with Formal Anonymity Guarantees, G. Fanti, S.B. Venkatakrishnan, S. Bakshi, B. Denby, S. Bhargava, A. Miller, and P. Viswanath, arXiv:1805.11060v1, May 28, 2018, [arxiv.org/pdf/1805.11060.pdf](https://arxiv.org/pdf/1805.11060.pdf)  
+[4] Chainalysis, [chainalysis.com](https://www.chainalysis.com/).
+[5] Adding Dandelion++ support to public networks: #6314, [github.com/monero-project/monero/pull/6314](https://github.com/monero-project/monero/pull/6314).  
+
+### Resumen
+
+Un poderoso método para prevenir la desanonimización en Monero está por venir, Dandelion++. Este método desarrollado originalmente para Bitcoin y aplicado a Monero por el desarrollador Lee Clagett, previene que adversarios conecten las direcciones IP a las transacciones de Monero, incluso en ataques sofisticados a gran escala. Se enfoca particularmente en la resistencia a botnets. Dandelion++ usa las fases de tallo y cipsela haciendo que la información tome forma de diente de león o dandelion en inglés, y por eso su nombre. Esta nueva característica mejorará las ya existentes características de privacidad de Monero y llevar a Moneroen un camino de progreso continuo. 
```

Comment:
y llevar a Moneroen un camino de progreso continuo..... "llevar a Monero en"

Reply:
Fixed, thank you :)